/**:
  ros__parameters:
    lqr_config_file: ''
    max_steering_angle_deg: 20.0
    steering_auto_override_deg: 4.0
    steering_units_multiplier: 1.0
    pure_pursuit_min_dist_m: 3.0
    pure_pursuit_max_dist_m: 7.0
    pure_pursuit_k_vel_m_ms: 0.75
    use_tire_velocity: true
    odometry_timeout_sec: 0.1
    path_timeout_sec: 0.5
    steering_max_ddeg_dt: 3.0
    invert_steering: false
